packages/react/src/primitives/View/View.tsx

```diff
@@ -27,6 +27,7 @@ const ViewPrimitive = <Element extends ElementType = 'div'>(
       'aria-label': ariaLabel,
       'data-testid': testId,
       disabled: isDisabled,
+      isDisabled: isDisabled,
```

Comment:
Why is this needed?

Reply:
this was added in because of the `FieldGroupIconButton` which is using the `as` property to render our primitive `Button` but the `isDisabled` flag was not being propagated down to the Button primitive.